.github/workflows/CodeQuality.yaml

```
@@ -0,0 +1,41 @@
+      uses: actions/checkout@v2
+    - name: Install dependencies
+      run: npm install
+    - name: Run e2e tests
+      run: npm run e2e-test
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: node:22
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)